src/main/docs/guide/introduction.adoc

```diff
@@ -32,13 +32,39 @@ This is achieved through a set of annotations:
 
 These annotations use internal Micronaut features and do not mock any part of Micronaut itself. When you run a test within `@MicronautTest` it is running your real application.
 
-In some tests you may need a reference to the `ApplicationContext` and/or the `EmbeddedServer` (for example, to create an instance of an `HttpClient`). Rather than defining these as properties of the test (such as a `@Shared` property in Spock), when using `@MicronautTest` you can reference the server/context that was started up for you, and inject them directly in your test.  
+In some tests you may need a reference to the `ApplicationContext` and/or the `EmbeddedServer` (for example, to create an instance of an `HttpClient`). Rather than defining these as properties of the test (such as a `@Shared` property in Spock), when using `@MicronautTest` you can reference the server/context that was started up for you, and inject them directly in your test.
 
 [source,groovy]
 ----
-@Inject 
+@Inject
 EmbeddedServer server //refers to the server that was started up for this test suite
 
-@Inject 
+@Inject
 ApplicationContext context //refers to the current application context within the scope of the test
+----
+
+### Eager Singleton Initialization
+
+If you enable https://docs.micronaut.io/latest/guide/index.html#eagerInit[eager singleton initialization] in your application, Micronaut will eagerly initialize all singletons at startup time. This can be useful for applications that need to perform some initialization at startup time, such as registering a bean with a third party library.
```

Comment:
```suggestion
If you enable https://docs.micronaut.io/latest/guide/index.html#eagerInit[eager singleton initialization] in your application, Micronaut Framework initializes eagerly all singletons at startup time. This can be useful for applications that need to perform some initialization at startup time, such as registering a bean with a third party library.
```

Reply:
"eagerly initializes", pushed here https://github.com/micronaut-projects/micronaut-test/pull/716/commits/f494c6638b9fc80985ce297f5fb688aadc5dbf52 🤞

Reply:
"eagerly initialises" 🇬🇧 

😜